OKR.md

```diff
@@ -0,0 +1,52 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q4
+
+**go-ipfs handles large datasets (1TB++) without a sweat**
+- `PX` - It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+- `PX` - It takes less than 12 hours to transfer 200GB sharded dataset over Fast Ethernet (100Mbps)
```

Comment:
Love these as KRs, but we should separately track the work we will do to achieve these key results. Is this faster bitswap?

Reply:
I believe that smarter bitswap is part of the way there, but there will be other parts.

Reply:
Steven says the second metric isn't testing something useful - he proposes:
- List a sharded directory with 1M entries in 10 mins

Reply:
How about:

- List a sharded directory with 1M entries over LAN in under 1 minute, with less than a second to the first entry.

Reply:
(replacement for the second KR - keep the first)